packages/plot/src/root.jsx

```diff
@@ -66,14 +64,16 @@ export class Root extends React.Component {
   }
 
   render() {
-    const { graphProps, children, classes, title, rootRef, paddingLeft } = this.props;
-    const { size, domain } = graphProps;
-    const padding = 50;
-    const longestWord = findLongestWord(domain.axisLabel);
-    const increaseWidth = amountToIncreaseWidth(longestWord);
-
-    const finalWidth = size.width + padding * 2 + increaseWidth;
-    const finalHeight = size.height + padding * 2;
+    const { graphProps, children, classes, title, rootRef } = this.props;
+    const {
+      size: { width, height },
+      domain,
+      range
+    } = graphProps;
+    const topPadding = 50;
+    const leftPadding = topPadding + 10; // left side requires an extra padding of 10
+    const finalWidth = width + leftPadding * 2 + domain.padding * 2;
+    const finalHeight = height + topPadding * 2 + range.padding * 2;
```

Comment:
Is there a chance domain.padding/ range.padding get here as undefined? If yes, then make sure you initialize them with 0/1 (which one works for you).